pybaseball/datahelpers/caching/cache_type.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum, unique
+
+@unique
+class CacheType(Enum):
+    CSV = 'csv'
+    CSV_BZ2 = 'csv.bz2'
+    CSV_GZ = 'csv.gz'
+    CSV_XZ = 'csv.xz'
+    CSV_ZIP = 'csv.zip'
+    FEATHER = 'feather'
+    FEATHER_LZ4 = 'lz4.feather'
+    FEATHER_UNCOMPRESSED = 'uc.feather'
+    FEATHER_ZSTD = 'zstd.feather'
+    JSON = 'json'
+    JSON_BZ2 = 'json.bz2'
+    JSON_GZ = 'json.gz'
+    JSON_XZ = 'json.xz'
+    JSON_ZIP = 'json.zip'
+    PARQUET = 'parquet'
+    PARQUET_GZ = 'gz.parquet'
+    PARQUET_FAST = 'fast.parquet'
+    PARQUET_FAST_GZ = 'fast.gz.parquet'
+    PICKLE = 'pickle'
+    PICKLE_BZIP = 'pickle.bzip'
+    PICKLE_GZ = 'pickle.gz'
+    PICKLE_XZ = 'pickle.xz'
+    PICKLE_ZIP = 'pickle.zip'
```

Comment:
Unnecessary. Maybe good for evaluation, but as I said, let's take our limited set and roll with it.

Reply:
> Unnecessary. Maybe good for evaluation, but as I said, let's take our limited set and roll with it.

Yeah I agree. To be clear this was just to evaluate our options. I didn't intend to merge with all these in place. 😀

Reply:
This has now been cut back to CSV, PARQUET, and PICKLE with PARQUET as the default